### ISSUE TITLE
No such command 'fill-curated-transformer'

### ISSUE THREAD

**Issue by DanShatford** [explosion/spaCy#13151]
I get the error `No such command 'fill-curated-transformer'.` when I try to run [`spacy init fill-curated-transformer`](https://spacy.io/api/cli#init-fill-curated-transformer).

Thank you for a great product and for your assistance in advance.

## How to reproduce the behaviour
1. Create a new python environment
2. `python -m pip install spacy spacy-curated-transformers`
3. `python -m spacy init fill-curated-transformer config.cfg - --model-name prajjwal1/bert-tiny`
```
Usage: python -m spacy init [OPTIONS] COMMAND [ARGS]...
Try 'python -m spacy init --help' for help.

Error: No such command 'fill-curated-transformer'.
```

## Your Environment
- **spaCy version:** 3.7.2
- **Platform:** macOS-14.1.1-x86_64-i386-64bit
- **Python version:** 3.10.11

**Comment by svlandeg** [comment]
Hi, thanks for the report! You're right - it looks like we didn't actually release this officially yet, though we have mentioned it in spaCy's documentation already. We'll get this fixed!

**Comment by adrianeboyd** [comment]
This has now been released with `spacy-curated-transformer` v0.2.1. Thanks for the PR!
